datakyt/db/dump_db.py

```diff
@@ -0,0 +1,47 @@
+import subprocess
+import logging
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG,
+                    format='%(asctime)s.%(msecs)03d[%(process)d:%(thread)d:%(name)s:%(lineno)d] %(levelname)s %('
+                           'message)s',
+                    datefmt='%Y-%m-%d %I:%M:%S')
+
+
+def backup_postgres_db(host, database_name, port, user, password, backup_file_path):
+    """
+  Backup postgres db to a file.
+
+   Parameters
+   ----------
+   host: the host to connect to (default localhost).
+   database_name: the name of database.
+   port: the port to connect on (default 5432).
+   user: the user account to connect as (default admin).
```

Comment:
As for now, the function doesn't have any default parameter values.  Add default values or correct the description

Reply:
I will remove all brackets in the description